### PR TITLE
[JA various phoemizers] PhoneticHint support and Unicode countermeasures

### PR DESCRIPTION
**Pull request by maiko3tattun**
- Support PhoneticHint : No conversion, but suffix and color are applied
- Replace NFD(か゛) with NFC(が) when entered in Unicode environment
- JA VCV & CVVC supports altarnate

### CODE REVIEW

**Comment by stakira** [comment]
Could you add tests to show your change at work?

File: OpenUtau.Plugin.Builtin/JapaneseCVVCPhonemizer.cs [hunk 1]
```diff
@@ -118,8 +118,11 @@ public class JapaneseCVVCPhonemizer : Phonemizer {
         // can probably be cleaned up more but i have work in the morning. have fun.
         public override Result Process(Note[] notes, Note? prev, Note? next, Note? prevNeighbour, Note? nextNeighbour, Note[] prevNeighbours) {
             var note = notes[0];
-            var currentUnicode = ToUnicodeElements(note.lyric);
```
Comment (stakira) [inline OpenUtau.Plugin.Builtin/JapaneseCVVCPhonemizer.cs:121]: Is `ToUnicodeElements()` no longer needed? How so?

**Comment by maiko3tattun** [comment]
#698 , #699 , #706 is merged.

File: OpenUtau.Plugin.Builtin/JapaneseCVVCPhonemizer.cs (hunk 1 above)
Comment (maiko3tattun) [reply OpenUtau.Plugin.Builtin/JapaneseCVVCPhonemizer.cs:121]: ToUnicodeElements() was only used to count characters correctly.
However, this approach does not allow が aliases to be called from か+゛.

Using Normalize() to remove surrogate characters, will allow correct character counts while resolving differences in string formatting.

**Comment by maiko3tattun** [comment]
I'm creating a test, so please wait a moment...

**Comment by maiko3tattun** [comment]
ok, added tests for Unicode to the three Japanese phoenemizers.

**Pull request merged**